src/java/boa/types/proto/MethodProtoTuple.java

```diff
@@ -1,6 +1,7 @@
 /*
- * Copyright 2014, Hridesh Rajan, Robert Dyer, 
```

Comment:
This file no longer needs changes.  Please revert all changes so this file is not listed as changed.